must-gather/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/openshift/origin-must-gather:4.3.0 as builder
+
+FROM centos:7
+
+# For gathering data from nodes
+RUN yum update -y && yum install iproute tcpdump pciutils util-linux nftables rsync -y && yum clean all
```

Comment:
for what do we need all these tools?

Reply:
removed,

Reply:
actually without rsync we get the following warning on running oc must-gather

```
WARNING: cannot use rsync: rsync not available in container
```